### DISCUSSION TITLE
[4.1] media storage folder

### DISCUSSION THREAD

**Discussion by HDInfautre** [joomla/joomla-cms#36118] [2. Cold Features]
Currently /images/
I would like to have (same level of the tree) another folder (new name)
Wish /0medias/
I don't have any other extensions that would cause problems and it doesn't seem to work.
Is it possible to fix it if it's a bug?
[I'm taking the liberty of making a request here, as I'm not a developer and I don't have a technical proposal to make. If I am not in the right place, please move or tell me how to do it].

**Comment by brianteeman** [comment]
You can do this with the core plugin FileSystem - Local

**Comment by HDInfautre** [comment]
Hi
Thank you for your support.
it doesn't work the way I think it does.
My request
To have a folder at the top of the tree (where I will put all my files (or at least all media: pdf, odt, images ....
/0_medias (or 0,medias same problem)
-/fr-doc
-/fr-img-art
-/img
...
/administrator
...
/images

Media under images are well rendered but not under the local path !!
See:
https://joom4test.88h.ovh/index.php?option=com_content&view=article&id=22:vrai-img&catid=2&Itemid=306
https://joom4test.88h.ovh/index.php?option=com_content&view=article&id=23:insert-img-1272-1920&catid=2&Itemid=306

and the images

![FireShot Capture 071 - Plugins _ Système de fichier - local - jooma4 test - Administration_ - joom4test 88h ovh](https://user-images.githubusercontent.com/6817936/143904907-deb57e43-d1a0-42d0-9b4a-48e50b4eba84.jpg)



![FireShot Capture 074 - Articles _ Modifier - jooma4 test - Administration - joom4test 88h ovh](https://user-images.githubusercontent.com/6817936/143904934-d13b13d7-3fea-404a-aa27-e6f3a141b580.jpg)

![FireShot Capture 077 - Articles _ Modifier - jooma4 test - Administration - joom4test 88h ovh](https://user-images.githubusercontent.com/6817936/143904959-37922617-f89c-41d0-a714-a61691f74ec7.jpg)

**Comment by HDInfautre** [comment]
Hi
An idea to solve this problem (setting, bug, misinterpretation) from someone? thank you
